docs/website/src/main/resources/certifications/jakarta-full-profile/9.0/TCK-Results.md

```diff
@@ -0,0 +1,595 @@
+TCK Results
+===========
+
+** These are Preliminary Results **
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta EE Platform, Full Profile.
+
+# Eclipse GlassFish 6.0, Jakarta EE 9 TCK Certification Summary
+
+- Product Name, Version and download URL (if applicable): <br/>
+  [Eclipse GlassFish 6.0](https://eclipse-ee4j.github.io/glassfish/download)
+  (https://download.eclipse.org/ee4j/glassfish/weekly/glassfish-6.0.0-SNAPSHOT-2020-10-25.zip)
+- Specification Name, Version and download URL: <br/>
+  [Jakarta EE Platform, 9.0](https://jakarta.ee/specifications/platform/9/)
+- TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta EE Platform TCK 9.0](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-jakartaeetck-9.0.0.zip), SHA-256: `661752b5c6c2e058b9ddb21b1ffbb36f9e99ad11fd51b8aa61bd10d6c235d308`
+- Public URL of TCK Results Summary: <br/>
+  [TCK results summary](TCK-Results.html)
+- Any Additional Specification Certification Requirements: <br/>
+  Jakarta Dependency Injection 2.0 TCK <br/>
+  Jakarta Contexts and Dependency Injection 3.0 TCK <br/>
+  Jakarta Bean Validation 3.0 TCK <br/>
+  Jakarta Batch 2.0 TCK tests are included in Jakarta EE 9 TCK tests, See Stage Name: jbatch <br/>
```

Comment:
Do we need jbatch as the additonal requirements?  
jbatch is integerated in the platform TCK.

Reply:
I believe Spec. says it is required. This clarifies how that requirement is met. We said the same thing in EE 8. I don't think this hurts anything. If you do think this causes confusion, I can remove it.

Reply:
I'm confused.
I thought this section describes the list of specifications which TCKs are not included the requested specification TCK.
If all specifications need to be listed as you mentioned, EJB, Servlet, JSP, and others must be also listed.
Maybe I am wrong.

Reply:
In general, all the stand-alone TCKs are actually run against Eclipse GlassFish. However, this TCK summary only lists the requirements for the Jakarta EE TCK. The Jakarta EE TCK rules list:

>     Jakarta Contexts and Dependency Injection 3.0
>     Jakarta Dependency Injection 2.0
>     Jakarta Bean Validation 3.0

I removed the reference to Batch.
Also -- I created a 9.0-RC1 sub-folder so there would be a permanent record for Specs. that will go to ballot ahead of the final EE 9 package (Interceptors, Enterprise Beans, and Managed Beans).

Reply:
Also, updated links and SHA codes for Jakarta EE TCK 9.0-RC1 and GlassFish 6.0 RC2